src/Native/System.Native/pal_interfaceaddresses.cpp

```diff
@@ -42,67 +42,71 @@ extern "C" int32_t EnumerateInterfaceAddresses(IPv4AddressFound onIpv4Found,
         int family = current->ifa_addr->sa_family;
         if (family == AF_INET)
         {
-            // IP Address
-            IpAddressInfo iai = {
-                .InterfaceIndex = interfaceIndex, .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
-            };
-
-            sockaddr_in* sain = reinterpret_cast<sockaddr_in*>(current->ifa_addr);
-            memcpy(iai.AddressBytes, &sain->sin_addr.s_addr, sizeof(sain->sin_addr.s_addr));
-
-            // Net Mask
-            IpAddressInfo maskInfo = {
-                .InterfaceIndex = interfaceIndex, .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
-            };
-
-            sockaddr_in* mask_sain = reinterpret_cast<sockaddr_in*>(current->ifa_netmask);
-            memcpy(maskInfo.AddressBytes, &mask_sain->sin_addr.s_addr, sizeof(mask_sain->sin_addr.s_addr));
-
-            onIpv4Found(current->ifa_name, &iai, &maskInfo);
+            if (onIpv4Found != nullptr)
```

Comment:
You can add ```?w=1``` to the query string to have GitHub hide whitespace differences (though annoyingly you can't then comment on it in that view).